Sources/Client/Client.cpp

```diff
@@ -56,6 +56,7 @@
 #include "NetClient.h"
 
 DEFINE_SPADES_SETTING(cg_chatBeep, "1");
+DEFINE_SPADES_SETTING(cg_alerts_volume, "1");
```

Comment:
- Use camelCase (e.g., `cg_alertsVolume`) like other cvars.
- I suggest something like `cg_alertSounds` since other cvars (`cg_chatBeep`, in particular) don't have "volume" in their names.